### PR TITLE
add: 📽️ Youtube playlists for learning foundry 

### PR DESCRIPTION
**Pull request by m0ham3dx**
Addition of youtube playlists for learning Foundry , beginner to advanced

### CODE REVIEW

File: src/tutorials/learn-foundry.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Foundry Tutorials (Unofficial) Videos
+
+Unofficial *youtube playlists* of Foundry tutorials from famousblockchain tutorial makers.
```
Comment (Evalir) [inline src/tutorials/learn-foundry.md:3]:
```suggestion
Unofficial *youtube playlists* of Foundry tutorials from Blockchain educators.
```

**Review by Evalir — CHANGES_REQUESTED** [review]
this looks good to me pending nit, wdyt @mds1 ?

File: src/tutorials/learn-foundry.md (hunk 1 above)
Comment (m0ham3dx) [reply src/tutorials/learn-foundry.md]: Thanks for the correction , i see you've already made the change so I dont have to make another PR to this

Comment (m0ham3dx) [reply src/tutorials/learn-foundry.md]: Ok thanks for already applying the changes, was a little confusing for me that I had to make another commit to this pr , and added the earlier message which i deleted

**Review by mattsse — APPROVED** [review]
nice

**Pull request merged**